platform-sdk/docs/hashgraph_glossary.md

```diff
@@ -0,0 +1,250 @@
+- __Self-descendent__`(x,y)` - a self-descendant of `x` is `y`. This means that `x` is a self-ancestor of `y`.
+- __See__`(x,y)` - `x` can see `y`. If there is no branching, then this is the same as `y` being an ancestor of `x`. If
+  there is branching, then it means that among all of `x` and its self-ancestors, `y` became an ancestor of that chain
+  earlier than any branch of `y` became an ancestor. In other words, you "see" your ancestors, and if there's a branch,
+  then you "see" the side of the branch that you became aware of first, and you never "see" the other side of that
+  branch, even after it becomes your ancestor.
+- __Strongly see__ (x,y) - `x` can strongly see `y`. This means that `x` can see events created by a supermajority of
+  nodes (`>2/3` by weight), each of which can see `y`. In other words, there are paths from `x` to `y`, following parent
+  pointers, that pass through a supermajority of creators. Because of the definition of "see", there will always be
+  paths through the creator of `x` and through the creator of `y`.
+- __Branch__(x,y) - `x` and `y` form a branch if all three of the following conditions hold. The definition of seeing
+  ensures that any given event can see at most one of `x` and `y`, if they form a branch. And for an honest node, if it
+  creates an event that sees one of `x` or `y`, all of its self-descendents will continue to see that same `x` or `y`,
+  and none of them will ever see the other one. The 3 requirements for them to constitute a branch are:
+    - both are created by the same creator
+    - neither is a self-ancestor of the other
+    - the difference of the birth rounds is at most `roundsAncient` (for the value of `roundsAncient` defined in the
+      roster whose roster round equals the max of their birth rounds)
+
+  [top](#top)
+
+<a name="generations"/>
+
+## Generations
+
+- __Local__ generation - an event is assigned this immutable number when it is added to the hashgraph. It is 1 plus the
+  max of all the local generations of its non-ancient parents. If it has no non-ancient parents, then it has a local
+  generation of 1. For a given event, this number can be different on different nodes. It can even be different on a
+  single node, before and after a restart or reconnect. The purpose of this number is to make it easy to sort a set of
+  events into topological order. (A topological order is any ordering of the events such that an event's parents always
+  precede it in the order).
+- __Consensus__ generation - an event is assigned this immutable number when it reaches consensus. It is 1 plus the max
+  of all the consensus generations of its eligible parents, where where an event is eligible if it is not ancient and
+  has not reached consensus in a previous round. If an event has no eligible parents, then its consensus generation is
+  1. This is deterministic, so an event is guaranteed to have the same consensus generation on all nodes. This is used
```

Comment:
`its consensus generation is` - is what?

And the numbered item that follows looks a bit out of context.